src/modules/organisations/services/organisations.service.ts

```diff
@@ -167,6 +168,7 @@ export class OrganisationsService implements OnApplicationBootstrap {
 				primary_site: organization.primary_site,
 				// Remark here organization is with Z
 				haorg_organization_type: organization?.sector || null,
+				form_url: isEmpty(organization?.form_url) ? null : organization?.form_url,
```

Comment:
We don't really need a lodash function to have a default value for empty strings:
```suggestion
				form_url: organization?.form_url || null,
```